src/controls/ScottPlot.Eto/ScottPlot.Eto.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <LangVersion>latest</LangVersion>
+        <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Eto.Forms" Version="2.5.11" />
```

Comment:
Does this mean this control is restricted to Windows Forms? I know I asked that question earlier but I wasn't sure if your answer was restricted to the demo app, not the control.

As an aside, it would probably be beneficial to have the demo target something cross-platform (or distribute multiple builds of it), but I would shy away from something like GTK as it's not installed by default on Windows. The more people can click and run the demo the better, in my opinion.

Reply:
The control is not restricted to WinForms - it is ready to be used by all Eto platforms.
The demo app is also ready to go for all Eto platforms - but needs to be built specifically for each.

In GitHub Actions it should be possible to do it with something like this ;-)
![image](https://user-images.githubusercontent.com/44544090/142238141-9da817c2-9c0f-468e-8001-56182a0fa400.png)
There are probably details to be addressed in creating those executables, but it should definitely be doable

Reply:
This is maybe preferable
![image](https://user-images.githubusercontent.com/44544090/142262667-62ca78c7-004b-4dc1-af26-1ea15b0a09ed.png)

Reply:
WinForms on the left - Wpf to the right
![image](https://user-images.githubusercontent.com/44544090/142262987-431c5159-f7d3-4941-bf72-8361d18e831e.png)
Wpf-buttons in Eto will not wrap text so looks bad (It might be doable but at a minimum requires some extra tweaking)